helm/auto-deployment/bold-bi-auto-deployment-using-helm.md

```diff
@@ -0,0 +1,439 @@
+# Bold BI Auto Deployment using Helm
+This section helps you to deploy [Bold BI](https://www.boldbi.com/) using helm chart without manually activating license and configuring the startup from browser. You can pass the license, startup and branding related customization as environment variable in values.yaml file.
```

Comment:
This section helps you to deploy [Bold BI](https://www.boldbi.com/) using the helm chart without manually activating the license and configuring the startup from the browser. You can pass the license, startup, and branding-related customization as environment variables in the values.yaml file.

Reply:
concern addressed